src/utils/debug.js

```diff
@@ -0,0 +1,33 @@
+function shouldDebug () {
```

Comment:
Why not write this file in `TS`?

Reply:
Because it cannot find the `window` type. Since this whole code is 20 lines I was lazy to find how to configure TS to accept both browser and Node global :sweat_smile: